src/interface/interface.json

```diff
@@ -415,6 +415,89 @@
                         "value": "{String} (optional) input value for text field",
                         "instructions": "{String} (optional) instructions under the text field"
                     }
+                },
+                "Range":{
```

Comment:
I wonder if we should call these methods `setMin`, `setMax`, and `setStep`. In other cases our properties and setters correspond when applicable. E.g. `Button.setTitle` and `TextField.setValue`. Our methods that attach event listeners correspond to html properties too. 

Seems to me we should stick to standard attribute names when they exist unless there is some reason to deviate.